ci-operator/jobs/openshift/hypershift/openshift-hypershift-release-4.16-periodics.yaml

```diff
@@ -787,6 +787,87 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
```

Comment:
can you add 
```
  reporter_config:
    slack:
      channel: '#hypershift-kubevirt-platform'
      job_states_to_report:
      - failure
      report_template: ':volcano: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
        <{{.Status.URL}}|View logs> :volcano:'
```

to the new generated jobs. That gives these periodics visibility in our slack channel.

Reply:
Correct, it was only to verify the non MCE flow on 4.14.
Let's drop it.

Reply:
done